scripts/fobnail-mount.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description="Mount Fobnail-protected disk"
+After=network.target
+StopWhenUnneeded=yes
+
+[Service]
+Type=simple
+RemainAfterExit=yes
+ExecStart=/hdd/projekty/fobnail/script-test/mount.sh
+ExecStop=/hdd/projekty/fobnail/script-test/umount.sh
```

Comment:
These also should be moved to another path. There is no way to parametrize these through configuration file, they have to be full, absolute paths, right?

Reply:
According to  using `EnvironmentFile` it is possible to variables in bash way, e.g. `ExecStart=${SOME_VARIABLE}`, however I didn't test this, I put the scripts in `/usr/share/fobnail`